### ISSUE TITLE
People are volunteering and receiving no response from their coordinators

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1119]
These contact messages were received in the past week

Hello I signed up to FreeCen to volunteer for transcribing just over a week ago and I haven't heard anything yet from a co-ordinator. I didn't specify a particular syndicate as I'm happy to work on anything. I did some transcription for FreeCen over 10 years ago and for FreeBMD about 20 years ago. Can you please let me know what I need to do to get started again? Thanks Nick Baggott

Good morning, I signed up to become a volunteer around a fortnight ago but have received no communication since sorting my log in details. I believe that I am still waiting to be contacted by a coordinator to give me my tasks. I look forward to hearing back from you so I can begin contributing. Many thanks, Georgia

Filled in an application to become a subscriber a few days ago but got no reply. So I applied again with some apparent conflict. I have been with FreeBMD for 15 years and have used name/password as - REDACTED user details.

E6 U6 S10 C5: P27

**Comment by FreecenBren** [comment]
That’s strange Kirk.
 
Nick is one of John Toplis  new volunteer, and I transferred him to LEICS at Johns request.
 
If the Georgia one is a Haney then I have her down on my list.
 
There is a John Hartley signed up for England, so I would have passed him on to another COORD.
He is not on my list.
 
I have 20 to set up myself for my counties.
 
If I have a 48 hour day I might get there.
 
We could do with the ‘signing on a person’ to take the pressure off the Coords.
I did mention that to Denise.
 
Brenda
